app/lib/directory.rb

```diff
@@ -102,18 +108,18 @@ def self.find_or_create(ldap_uid)
   # Net::LDAP::Entry.
   def self.search_ldap(term)
```

Comment:
Because Directory was changed to be used as an initialized object all of the class methods in this file probably need to be changed to instance methods.